app/models/global_settings/boolean_value.rb

```diff
@@ -1,4 +1,4 @@
-class Instance::Settings::BooleanValue
+class GlobalSettings::BooleanValue
```

Comment:
This can just be global, I think.

Otherwise you can call it Pseudo::BooleanValue since it isn't really one.